tests/ft_rust.vader

```diff
@@ -1,5 +1,11 @@
 Include: include/setup.vader
 
+Before (create src directory):
+  call mkdir('src')
+
+After (cleanup src directory):
+  !rmdir src
+
```

Comment:
As for the `bufnr`, Vim might create a new buffer now?
You can add `Log neomake#utils#redir('ls!')` to see the buffer list.

"More than 1 window after tests" is because of the previous failure.